graph_model/model.py

```diff
@@ -0,0 +1,35 @@
+import graph_model.trie_model as trie_model
+import json
+class GraphModel:
+    def __init__(self):
+        self.trie = trie_model.Trie()
+
+    def train(self):
+        try:
+            data_file = open('./temp/data/pre_proceed_data.txt', 'r')
```

Comment:
Can place files to the top level in temp, there is not too many of them. Otherwise, there is a file named data in data directory.